src/components/Button/Button-story.js

```diff
@@ -12,29 +12,29 @@ import { withKnobs, boolean, select } from '@storybook/addon-knobs';
 import { iconAddSolid, iconSearch } from 'carbon-icons';
 import AddFilled16 from '@carbon/icons-react/lib/add--filled/16';
 import Search16 from '@carbon/icons-react/lib/search/16';
-import { settings } from 'carbon-components';
 import Button from '../Button';
 import Link from '../Link';
 import ButtonSkeleton from '../Button/Button.Skeleton';
-import { componentsX } from '../../internal/FeatureFlags';
-
-const { prefix } = settings;
+import { breakingChangesX } from '../../internal/FeatureFlags';
 
 const icons = {
   None: 'None',
-  'Add with filled circle (iconAddSolid from `carbon-icons`)': componentsX
-    ? 'AddFilled16'
-    : 'iconAddSolid',
-  'Search (iconSearch from `carbon-icons`)': componentsX
-    ? 'Search16'
-    : 'iconSearch',
+  ...(breakingChangesX
```

Comment:
For code style, instead of doing a ternary on spread, might be helpful to do a pattern similar to:

```js
const icons = {
  // ...
};

if (breakingChangesX) {
  icons['Add with ...'] = 'AddFilled16';
  // ...
}
```

Reply:
OK fair enough, made a change.